pkg/cmd/get/get_build_logs.go

```diff
@@ -490,36 +488,6 @@ func (o *CLILogWriter) BytesLimit() int {
 	return 0
 }
 
-func checkForStagePod(kubeClient kubernetes.Interface, ns string, pr *tektonv1alpha1.PipelineRun, pri *tekton.PipelineRunInfo, stage *tekton.StageInfo) error {
```

Comment:
why can this be removed?

Reply:
It was unused, so while I was here, I cleaned things up.